server/service/individualServices/SoftwareUpgrade.js

```diff
@@ -88,6 +89,7 @@ async function PromptForBequeathingDataCausesTransferOfListOfApplications(user,
                      *   /v1/regard-application
                      ************************************************************************************/
                     let requestBody = {};
+                    isRequired = true;
```

Comment:
isRequired is a very generic ter. Kindly give some complete name like "isResponseRequired"